### PR TITLE
Add Postgres upgrade test

### PR DESCRIPTION
**Pull request by c-du**
## Description

Add Postgres upgrade test
1. Test rocksdb -> Postgres upgrade
2. Bounce central

## Checklist
- [x] Investigated and inspected CI test results
- ~[ ] Unit test and regression tests added~
- ~[ ] Evaluated and added CHANGELOG entry if required~
- ~[ ] Determined and documented upgrade steps~
- ~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

If any of these don't apply, please comment below.

## Testing Performed

Run upgrade tests.

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

File: tests/upgrade/postgres_run.sh [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+    local log_output_dir="$1"
+
+    require_environment "KUBECONFIG"
+
+    export_test_environment
+
+    REPO_FOR_TIME_TRAVEL="/tmp/rox-postgres-upgrade-test"
+    DEPLOY_DIR="deploy/k8s"
+    QUAY_REPO="rhacs-eng"
+    REGISTRY="quay.io/$QUAY_REPO"
```
Comment (dashrews78) [inline tests/upgrade/postgres_run.sh:33]: Do we need to put the is_CI check here like the other version of run has?  I'm not sure these are being executed outside of CI though.

    if is_CI; then
        REGISTRY="quay.io/$QUAY_REPO"
    else
        REGISTRY="stackrox"
    fi

File: tests/upgrade/postgres_run.sh [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+    if [[ "$#" -ne 1 ]]; then
+        die "missing args. usage: test_upgrade_paths <log-output-dir>"
+    fi
+
+    local log_output_dir="$1"
+
+    EARLIER_SHA="9f82d2713cfec4b5c876d8dc0149f6d9cd70d349"
+    EARLIER_TAG="3.63.x-163-g2c4fe1563c"
+    #EARLIER_SHA="270d517e9572a8301a62e93d9299bd66c15ef4f8"
```
Comment (dashrews78) [inline tests/upgrade/postgres_run.sh:95]: nit:  remove these

**Review by dashrews78 — APPROVED** [review]
Looks pretty good to me.  just a couple of nits.

File: tests/upgrade/postgres_run.sh (hunk 1 above)
Comment (c-du) [reply tests/upgrade/postgres_run.sh:33]: We cannot use docker.io any more.

**Pull request merged**